orchagent/portsorch.h

```diff
@@ -93,6 +94,8 @@ class PortsOrch : public Orch, public Subject
     bool addSubPort(Port &port, const string &alias, const bool &adminUp = true, const uint32_t &mtu = 0);
     bool removeSubPort(const string &alias);
 private:
+    std::vector<Port> m_portsToAdd;
```

Comment:
>vector [](start = 9, length = 6)

#include <vector>

Reply:
Included in `port.h` and already used on [#L22](https://github.com/Azure/sonic-swss/blob/master/orchagent/portsorch.h#L22)